docs/platforms/unity/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Automatic Instrumentation
+description: "Learn what transactions are captured after tracing is enabled."
+sidebar_order: 20
+---
+
+## Automatically Captured Transactions
+
+The Unity SDK automatically creates transactions for the game's startup and scene loading. Additionally, if opted-in, it generates spans for all `Awake` calls during these operations.
+
+You can manage these settings through the Editor Config Window or <Link to="/platforms/unity/configuration/options/#programmatic-configuration">programmatically</Link>.
```

Comment:
```suggestion
The Unity SDK automatically creates transactions for the game's startup and scene loading. 

If you want the SDK to also generate spans for all `Awake` calls during startup and scene operations, you can enable the following settings through the Editor Config Window or <Link to="/platforms/unity/configuration/options/#programmatic-configuration">programmatically</Link>.
```

Clarify if the settings below are to generate spans for Awake calls or just for scene loading and startup. I assume it's for awake calls since you said startup & scene loading transactions are automatically captured, but it might be confusing.